app/ui.js

```diff
@@ -436,6 +443,10 @@ var UI;
                     break;
                 case 'disconnected':
                     UI.showStatus(_("Disconnected"));
+
+                    if(UI.reconnect) {
+                        setTimeout(UI.connect, UI.reconnect_delay);
```

Comment:
What do you think about displaying "reconnecting" instead of "connecting" in the status message here?